js/apps/events-page/controllers/eventsPageCtrl.js

```diff
@@ -14,6 +14,10 @@
 		metaService,
 		RequestModel
 	) {
+        setTimeout(() => {
```

Comment:
You probably already tried it, but would $document.ready() work here?

Reply:
Also, does the "pop" the window into place? You could always trigger the scrollToTop button instead, which is a smooth scrolling effect.

Reply:
Adding a stupid comment so I can stupid request changes.

Reply:
i wonder if window.onload might work here

Reply:
I tried all of these without success. Any other thoughts?

Reply:
I guess we're going with setTimeout then. Worse things have happened.

Reply:
☹️